src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -137,11 +144,23 @@ const TableWrapper = (props: TableWrapperProps) => {
           <TableBodyWrapper {...props} setShouldRefocus={setShouldRefocus} tableWrapperRef={tableWrapperRef} />
         </StyledTable>
       </StyledTableContainer>
-      {interactions.active && (
-        <FooterWrapper footerContainer={footerContainer} paginationNeeded={paginationNeeded}>
-          <PaginationContent {...props} handleChangePage={handleChangePage} isSelectionMode={isSelectionMode} />
-        </FooterWrapper>
-      )}
+      <PaginationFooter
+        footerContainer={footerContainer}
+        pageInfo={pageInfo}
+        paginationNeeded={paginationNeeded}
+        theme={theme}
+        totalRowCount={totalRowCount}
+        totalColumnCount={totalColumnCount}
+        totalPages={totalPages}
+        keyboard={keyboard}
+        translator={translator}
+        interactions={interactions}
+        rect={rect}
+        layout={layout as unknown as EngineAPI.IGenericHyperCubeLayout}
```

Comment:
Fairly sure I can remove the type casting now. Was able to do so in the pivo table

Reply:
It just needs the chart id.. So yes could since in just that TBH.

Type casting is needed in the sn-table becase `qHyperCube` is omitted (`interface TableLayout extends Omit<EngineAPI.IGenericHyperCubeLayout, "qHyperCube">`).

I won't change that in this PR. But will do a change in the util repo to send in just the data that is needed.

Reply:
great